### PR TITLE
basic support for passive-interface and static routes

### PR DESCRIPTION
**Pull request by clxnetom**
This is not fully tested yet but if it is usable (as in something to build on) no one is more happy than me.

If you have the time and find any misstakes, errors or just bad coding please let me know.

### CODE REVIEW

File: lib/puppet/provider/quagga_ospf_router.1/quagga.rb [hunk 1]
```diff
@@ -0,0 +1,243 @@
+Puppet::Type.type(:quagga_ospf_router).provide :quagga do
```
Comment (bibigon812) [inline lib/puppet/provider/quagga_ospf_router.1/quagga.rb]: Why do you create the new folder `quagga_ospf_router.1`?

Comment (bibigon812) [inline lib/puppet/provider/quagga_ospf_router.1/quagga.rb]: Can you make a new PR with the quagga_static_route resource type only?

**Comment by clxnetom** [comment]
OK, should be taken care of now.

**Comment by bibigon812** [comment]
I already implement the `passive_interfaces`.

**Pull request closed without merging**